lib/configure.js

```diff
@@ -1,10 +1,93 @@
 'use strict';
 
+const fs = require('fs');
+const path = require('path');
+
+const async = require('async');
+const _ = require('lodash');
 const glob = require('glob');
-const jsyaml = require('jsyaml');
+const jsyaml = require('js-yaml');
+
+
+module.exports = (functionList, opts, done) => {
+  
+  let envFiles = [];
+
+  async.auto({
+    getDefaultConfigs: (cb) => {
+      glob("conf/**/default*", (err, files) => {
+        if(err) {
+          return cb(err);
+        }
+
+        async.map(files, (file, acb) => {
+          acb(null, jsyaml.safeLoad( fs.readFileSync(file, 'utf-8') ));
+        }, cb);
+      }); 
+    },
+    getEnvConfigs: (cb) => {
+      glob("conf/**/!(default*)", (err, files) => {
```

Comment:
Yes.
Confi assumes that you are in an environment, so all the configurations are passed translated down into a single object.
The lambda function would need to know about the configurations for all the environments at the same time to choose which one it needs.